release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -3155,3 +3155,58 @@ $ oc adm release info 4.13.11 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-13-12"]
+=== RHBA-2023:5011 - {product-title} 4.13.12 bug fix
+
+Issued: 2023-09-12
+
+{product-title} release 4.13.12 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:5011[RHBA-2023:5011] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:5014[RHBA-2023:5014] advisory.
```

Comment:
FYI, `https://access.redhat.com/errata/RHBA-2023:5011` and `https://access.redhat.com/errata/RHBA-2023:5014` open a page not found.

Reply:
No worries, the release hasn't shipped yet. They will work once things ship. Thanks for the review.